docs/source/torch.rst

```diff
@@ -12,19 +12,21 @@ Tensors
 .. autofunction:: set_printoptions
 .. autofunction:: set_flush_denormal
 
+.. _tensor-creation-ops:
 
 Creation Ops
 ~~~~~~~~~~~~~~~~~~~~~~
-.. autofunction:: eye
+.. autofunction:: tensor
```

Comment:
Can we add some text here on how random sampling methods also create tensors and take in similar args (randn, rand, randint, randperm etc)